src/redux/store.ts

```diff
@@ -142,6 +142,7 @@ export const setupStore = (initialState = {}, config = persistConfig) => {
           'keylessBackup',
           'nfts',
           'swap',
+          // 'priceHistory'
```

Comment:
can you leave some indication of when this should be added? like //TODO(ACT-XYZ): uncomment this

Reply:
or if you have no immediate plans for uncommenting it, you can just delete this line..

Reply:
This won't be added it's just for local debugging similar to `tokens` and `exchange`.

Reply:
then please delete it

Reply:
if you feel strongly about keeping it please at least add a code comment explaining why it's there, like 
```
// the following can be uncommented for debugging:
// 'priceHistory',
// 'tokens',
// 'exchange',
```

Reply:
otherwise it's not clear why it's there (was it used previously but commented out to *solve* a bug? is it something you meant to build on? or still plan to do something with?)

Reply:
Left more detailed comments in f83c36a6029bbbef1b4f3de05bd54772cf26e2fd!